### ISSUE TITLE
Add "Free Download Manager" App

### ISSUE THREAD

**Issue by Bellisario** [portapps/portapps#178]
> Please follow this guide to file a proper request report : https://portapps.io/doc/request-app/

* Name : Free Download Manager
* Description : An app to manage all downloads and download also videos and audio from most famous platforms
* Website : https://www.freedownloadmanager.org/
* License (e.g. Freeware, OSS, GPL, MIT, etc) : GPL
* Comment (anything else which might help) :

**Comment by kybo15** [comment]
The official application is already portable. open the options and you will find the option to make it portable.

**Comment by Bellisario** [comment]
Yes @kybo15, but doing this, the application creates its folder even it's portable, with portapps the application folder remains to the "portable" folder. Then, there is another problem: in your way, you need to install the application in a computer first, and then create the portable one.
